app/assets/stylesheets/provider/_flash.scss

```diff
@@ -1,7 +1,7 @@
 #flashWrapper {
   position: absolute;
-  right: 0;
-  top: line-height-times(0.5);
+  right: line-height-times(1);
+  top: line-height-times(1);
```

Comment:
this might be a problem on other pages than the dashboard where it might overlap header i think

Reply:
Since it's positioned relative to "wrapper", I don't think that won't be an issue. This is an example in Analytics:
![Screen Shot 2019-06-20 at 15 13 52](https://user-images.githubusercontent.com/11672286/59851909-05881200-936e-11e9-90a3-84fcdf3d7e79.png)

Reply:
<img width="1211" alt="Screenshot 2019-06-20 15 27 41" src="https://user-images.githubusercontent.com/54224/59852878-0752d500-9370-11e9-90f4-d62edc33badc.png">

I think it wouldn't fit here for example

Reply:
So the problem is not overlapping the header but the horizontal nav? It can certainly be half-line, no problem about that. I thought you meant something else.

This is how it looks with 1 line top and right, anyways:
![Screen Shot 2019-06-20 at 15 35 11](https://user-images.githubusercontent.com/11672286/59853399-18e8ac80-9371-11e9-924d-d8e0f2bc1fde.png)